Core/GDCore/Extensions/Builtin/SpriteExtension/SpriteExtension.cpp

```diff
@@ -595,33 +631,14 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsSpriteExtension(
                     "res/actions/sprite.png")
       .AddParameter("object", _("Object"), "Sprite");
 
-  obj.AddExpression("AnimationSpeedScale",
-                    _("Animation speed scale"),
-                    _("Animation speed scale"),
-                    _("Animations and images"),
-                    "res/actions/animation.png")
-      .AddParameter("object", _("Object"), "Sprite");
-
-  obj.AddExpression("ScaleX",
-                    _("Scale of the width of an object"),
-                    _("Scale of the width of an object"),
-                    _("Size"),
-                    "res/actions/scaleWidth_black.png")
-      .AddParameter("object", _("Object"), "Sprite");
-
-  obj.AddExpression("ScaleY",
```

Comment:
Should you keep those expressions for backwards compatibility?

Reply:
Yes, I think I removed it when working on the previous solution and forgot to revert this.